app/models/concerns/authenticatable.rb

```diff
@@ -0,0 +1,16 @@
+module Authenticatable
```

Comment:
This is just basic Devise stuff, decoupling the fact that this represents authentication concerns from the actual implementation of `app/models/user.rb`. If we ever wanted to swap out the authentication backend, the `app/models/user.rb` file would experience zero code churn as a result.